open-craft/opencraft
# BB-3865 Add e2e tests for password reset

Adds e2e tests for forgot password page.
Adds e2e test for reset password page.

**Related Tickets**: [BB-3865](https://tasks.opencraft.com/browse/BB-3865)
**Github Issue**: [#716](https://gitlab.com/opencraft/dev/opencraft/-/issues/716)

**Testing Instruction**
1. Checkout to this branch
2. Check that the test covers possible actions in password reset flow
3. Run `make test.e2e` in vagrant box
4. Make sure all the tests pass
5. Make sure that the CI job `e2e_tests` pass.
